database/database_manager.py

```diff
@@ -88,7 +120,13 @@ def __init__(self,
             # Create cursor, and execute the query.
             with connection.cursor() as cursor:
                 cursor.execute(query, values)
-                result = cursor.fetchall()
+                # Check if cursor.description is NONE - meaning no results returned.
+                if cursor.description:
+                    result = cursor.fetchall()
+                else:
+                    # Return a blank tuple if there are no results, since we are
+                    # forcing this to a list.
+                    result = tuple()
```

Comment:
```suggestion
                    result = ()
```
Tuple literal is more performant.

Reply:
fixed in a5f33f5.